controls/cis_ocp_1_4_0/section-4.yml

```diff
@@ -135,13 +137,15 @@ controls:
       status: automated
       rules:
         - kubelet_configure_tls_cert
+        - kubelet_configure_tls_key
       levels: level_1
     - id: 4.2.10
       title: Ensure that the --rotate-certificates argument is not set to false
       status: automated
       rules:
         - kubelet_enable_client_cert_rotation
         - kubelet_enable_cert_rotation
+        - controller_rotate_kubelet_server_certs
```

Comment:
This rule is dealing with `RotateKubeletServerCertificate` not 'RotateKubeletClientCertificate'.

And what is the difference between this rule and the one currently assigned to `4.2.11`: [kubelet_enable_server_cert_rotation](https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/kubelet/kubelet_enable_server_cert_rotation/rule.yml)

Reply:
if you look at the audit of the control, it has two parts:
```
Audit:
Run the following command to check if the RotateKubeletServerCertificate feature
gate is enabled on each node:

for node in $(oc get nodes -ojsonpath='{.items[*].metadata.name}'); do
oc get --raw /api/v1/nodes/$node/proxy/configz | jq
'.kubeletconfig.featureGates.RotateKubeletServerCertificate'
done
Verify that all nodes return true.


Run the following command to check that each node is configured to rotate certificates:
for node in $(oc get nodes -ojsonpath='{.items[*].metadata.name}'); do
oc get --raw /api/v1/nodes/$node/proxy/configz | jq
'.kubeletconfig.rotateCertificates
```

kubelet_enable_server_cert_rotation deals with the second check in the audit, the one this PR is adding deals with the first audit check (just in a slightly different manner, because the rule is old and we didn't know how to check the proxy/configz object back then)

Reply:
Thanks for the clarification